app-emulation/euae/euae-0.8.29.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="E-UAE Experimental Ubiquitous Amiga Emulator"
+DESCRIPTION="This is a version of UAE, the Ubiquitous Amiga Emulator, with an emulation core largely based on WinUAE. It attempts to bring many of the great features of WinUAE to non-Windows platforms. This version now finally has a name, E-UAE, since that's what everybody was calling it anyway. The 'E' can stand for anything you fancy. Experimental, extreme, exciting, egalitarian, eggplant, . . .
```

Comment:
DESCRIPTION should be defined with lines of at most 80 chars, including the **`\`** used to fold the lines.
If you are unsure of where to fold the lines, don't worry. I can grab your remote branch, fix it on my local machine and push it back to your remote branch if you wish, provided you check the "[Allow edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)" checkbox.

Hmm, you are using master instead of [feature-branches](https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches)... Using master is not recommended but it is still workable.